keras/engine/topology.py

```diff
@@ -1236,6 +1236,10 @@ def get_output_shape_for(self, input_shape):
         elif self.mode == 'concat':
             output_shape = list(input_shapes[0])
             for shape in input_shapes[1:]:
+                if output_shape[self.concat_axis] is None or \
```

Comment:
`\` should be avoided. Use parens, or just cram everything on the same line (not such an unreasonable thing to do).